docs/modules/transform-sync/utilities.md

```diff
@@ -0,0 +1,39 @@
+<%(TOC max="4")%>
+
+# Transform utilities
+
+The transform synchronization module includes a [`TransformUtils`]({{urlRoot}}/api/transform-synchronization/transform-utils) class, populated with a set of static methods that makes it nicer to use the feature module.
+
+## Type conversion methods
+
+The following static methods can be used to construct a `FixedPointVector3` or `CompressedQuaternion` from their respective native Unity types.
+
+| Static Method                                            | Result Type            |
+|----------------------------------------------------------|------------------------|
+| `FixedPointVector3.FromUnityVector(Vector3 v)`           | `FixedPointVector3`    |
+| `CompressedQuaternion.FromUnityQuaternion(Quaternion q)` | `CompressedQuaternion` |
+
+The methods below can be called to easily convert a variable to and from native Unity types.
```

Comment:
```suggestion
The extension methods below can be called to easily convert a variable to and from native Unity types.
```

Maybe?

Reply:
I nearly wrote that, but technically not every one of them is an extension method